scripts/gen-shaping-tests.py

```diff
@@ -99,7 +99,7 @@ def convert_test(hb_dir, hb_shape_exe, tests_name, file_name, idx, data, fonts):
     glyphs_expected = subprocess.run(options_list, check=True, stdout=subprocess.PIPE)\
         .stdout.decode()
 
-    glyphs_expected = glyphs_expected[1:-2]  # remove `[..]\n`
+    glyphs_expected = glyphs_expected[:-1]  # remove trailing newline
```

Comment:
I think this one should stay as before. No need for needless `[]`

Reply:
This is the minor change to `gen-shaping-tests.py` I alluded to. :)

Reply:
It overwrites all the rust tests, which is better to avoid.